openslides/core/views.py

```diff
@@ -436,6 +451,40 @@ def update(self, request, *args, **kwargs):
         # Return response.
         return Response({"key": key, "value": value})
 
+    @list_route(methods=["post"])
+    def bulk_update(self, request):
+        """
+        Updates many config variables:
+        [{key: <key>, value: <value>}, ...]
+        """
+        if not isinstance(request.data, list):
+            raise ValidationError({"detail": "The data needs to be a list"})
+
+        for entry in request.data:
+            key = entry.get("key")
+            if not isinstance(key, str):
+                raise ValidationError({"detail": "The key must be a string."})
+            if not config.exists(key):
+                raise ValidationError(
+                    {"detail": "The key {0} does not exist.", "args": [key]}
+                )
+            if not self.check_config_permission(key):
+                self.permission_denied(request, message=key)
+            value = entry.get("value")
+            if value is None:
+                raise ValidationError(
+                    {"detail": "Invalid input. Config value is missing."}
+                )
+
+        errors = {}
+        for entry in request.data:
+            try:
+                config[entry["key"]] = entry["value"]
+            except ConfigError as err:
+                errors[key] = str(err)
```

Comment:
The last error overwrites the previous errors. So only one error is returned, even if several fields contain errors.

Reply:
Yes, it should be: errors[entry["key"]] = str(err)